src/anubhav/staticandnonstaticexamples/CandidateGrade.java

```diff
@@ -0,0 +1,87 @@
+package anubhav.staticandnonstaticexamples;
+
+/*
+Design to test static, non static, return type concepts.
+
+1) Method sum() - non static method, accept 3 parameters of type double & return answer
+
+2) Method average()- non static method, accept 3 parameters of type double & return answer
+
+3) Method isAnswerAboveExpecation() - non static method, take one parameter (answer which you received from average method) 
+	and return true if value is more then 50 else false.
+
+4) Method isEligible() - static method,take one parameter (answer which you received from average method) return true if 
+	value is more then 50 or divisible by 2. Eligibility will be printed in main method. 
+
+5) Method getYourGrade() - static method, take one parameter (answer which you received from average method) 
+	return "A Grade" if average is more then 80 else "B grade", main method will print grade received from getYourGrade() method.
+
+Sample input1 : 
+10,20,30
+Sample output1 : 
+Candidate is Eligible
+Candidate obtained B grade.
+
+Note : Sum method should not be called from main method, it should be called from Average method (think about code reusability).*/
+
+public class CandidateGrade {
+	
+	double sum (double n1, double n2, double n3) { // sum of numbers
+		double sumOfNumbers= (n1 + n2 +n3);
+		return sumOfNumbers;
+	}
+
+	double average (double n1, double n2, double n3) { // average of sum of numbers
+		double averageOfNumbers= (sum(n1, n2, n3)/3);
+		return averageOfNumbers;
+	}
+	
+	boolean isAnswerAboveExpectation (double avg) { //avg > 50 condition
```

Comment:
this method not getting called from anywhere?
then what is the use case of this?